conf/default-coverage.json

```diff
@@ -1,79 +1,26 @@
 {
-    "ignore": {
-        "clientside": {
-            "inapp": [
-                "/underscore.js",
-                "/meteor.js",
-                "/meteor-base.js",
-                "/mobile-experience.js",
-                "/babel-compiler.js",
-                "/ecmascript.js",
-                "/base64.js",
-                "/ejson.js",
-                "/id-map.js",
-                "/ordered-dict.js",
-                "/tracker.js",
-                "/modules-runtime.js",
-                "/modules.js",
-                "/es5-shim.js",
-                "/promise.js",
-                "/ecmascript-runtime.js",
-                "/babel-runtime.js",
-                "/random.js",
-                "/mongo-id.js",
-                "/diff-sequence.js",
-                "/geojson-utils.js",
-                "/minimongo.js",
-                "/check.js",
-                "/retry.js",
-                "/ddp-common.js",
-                "/reload.js",
-                "/ddp-client.js",
-                "/ddp.js",
-                "/ddp-server.js",
-                "/allow-deny.js",
-                "/insecure.js",
-                "/mongo.js",
-                "/blaze-html-templates.js",
-                "/reactive-var.js",
-                "/jquery.js",
-                "/standard-minifier-css.js",
-                "/standard-minifier-js.js",
-                "/autopublish.js",
-                "/dburles_eslint.js",
-                "/webapp.js",
-                "/livedata.js",
-                "/hot-code-push.js",
-                "/deps.js",
-                "/htmljs.js",
-                "/observe-sequence.js",
-                "/blaze.js",
-                "/spacebars.js",
-                "/templating.js",
-                "/launch-screen.js",
-                "/ui.js",
-                "/autoupdate.js",
-                "/global-imports.js",
-                "/lmieulet_meteor-coverage.js",
-                "/practicalmeteor_mocha.js",
-                "/practicalmeteor_loglevel.js",
-                "/practicalmeteor_sinon.js",
-                "/practicalmeteor_mocha-console-runner.js",
-                "/practicalmeteor_mocha-core.js"
-            ],
-            "public": [
-            ]
-        },
-        "serverside": [
-            "livedata.js",
-            "hot-code-push.js",
-            "launch-screen.js",
-            "templating.js",
-            "autoupdate.js",
-            "reload.js",
-            "global-imports.js"
+    "include": [
```

Comment:
I'm not enforcing it to be imported, I'm adding it as an exception for say, ignored folders. So if a filepath matches this pattern it is never ignored even if it match a `ignore` pattern rule